CONTRIBUTING.md

```diff
@@ -182,5 +183,8 @@ git config --local core.hooksPath .githooks
 A reminder for the maintainers on how to create a new release.
 
 1. Make sure all your changes are committed.
+1. Verify that [``CHANGELOG.md``](CHANGELOG.md) has all the relevant changes. Visit [this link](https://github.com/NLESC-JCER/cpp2wasm/compare/v0.1.0...master) to see the diff between a released version and ``master``.
```

Comment:
I don't like the `v0.1.0`, this will give wrong results when not updated. And updating the version here will not happen. Can we instead describe how to get to the link
```suggestion
1. Verify that [``CHANGELOG.md``](CHANGELOG.md) has all the relevant changes. Visit [releases page](https://github.com/NLESC-JCER/cpp2wasm/releases) and click on the `?? commits to master since this release` link in the latest release to see the diff between the latest release and ``master``.
```